### ISSUE TITLE
Click action "Toggle single / Preview multiple" no longer working in in Gnome 42.0 with v46

### ISSUE THREAD

**Issue by Janhouse** [home-sweet-gnome/dash-to-panel#1615]
**Linux distribution and version**
Archlinux

**GNOME Shell version**
GNOME Shell 42.0

**Dash-to-Panel version**
46

**Where was Dash-to-Panel installed from?**
The GNOME extensions website

I am using click action "Toggle single / Preview multiple" which shows window previews only after clicking on an application that has multiple windows open. It was working nicely until I upgraded to Gnome 42.

![image](https://user-images.githubusercontent.com/1036439/162485408-7e794a32-6953-4dae-8ada-1f324e7469ce.png)

Log shows this error when clicking on applications:
```
JS ERROR: TypeError: event.get_click_count is not a function
activate@/home/janhouse/.local/share/gnome-shell/extensions/dash-to-panel@jderose9.github.com/appIcons.js:957:43
vfunc_clicked@resource:///org/gnome/shell/ui/appDisplay.js:3162:14
```

**Comment by Janhouse** [comment]
Never mind, duplicate of #1611
